Tasks/BashV3/Tests/L0.ts

```diff
@@ -107,7 +107,7 @@ describe('Bash Suite', function () {
 
         runValidations(() => {
             assert(tr.failed, 'Bash should have failed');
-            assert(tr.stdout.indexOf('##vso[task.issue type=error;]myErrorTest') > 0, 'Bash should have correctly written myErrorTest');
+            assert(tr.stdout.indexOf('##vso[task.issue type=error;source=CustomerScript;]myErrorTest') > 0, 'Bash should have correctly written myErrorTest');
```

Comment:
Not directly related to this line, but if possible, we should try to attempt to prevent source from being specified from custom scripts (e.g. bash/powershell).

E.g if a batch script or bash script writes a log command with "source="; we should detect it and raise an error

likewise, if a non-internal task specifies "source=" in a logging command, it should be an error too

Error should be something like 'Specifing source argument in the logging command is not supported'

Reply:
We can implement this feature for the tasks based on the Node handler because the task-lib fully manages the subprocess (which runs a script or a program), its stdin, stdout, etc. It's easy to add the logic to the event handler on stdout data and process it.

But I didn't find the same for Vsts SDK (PowerShell). It runs the commands natively and synchronously using the `Invoke-Expression `command. Especially since PowerShell 3.0 is very limited, this will require big changes to the Vsts logic.